### PR TITLE
add Vietnamese

### PR DESCRIPTION
**Pull request by hieusats**

### CODE REVIEW

**Comment by kinaskitaro** [comment]
add vietnamese plz

**Comment by phamducvu** [comment]
add Vietnamese, thank you!

File: en.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ export const en = {
     fr: 'Français',
     es: 'Español',
     pt: 'Português',
+    vn: 'Vietnamese',
```
Comment (alexanvl) [inline en.js]: Language setting should be translated

File: es.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ export const es = {
     fr: 'Français',
     es: 'Español',
     pt: 'Português',
+    vn: 'Vietnamese',
```
Comment (alexanvl) [inline es.js]: Language setting should be translated

File: fr.js [hunk 3]
```diff
@@ -7,6 +7,7 @@ export const fr = {
     fr: 'Français',
     es: 'Español',
     pt: 'Português',
+    vn: 'Vietnamese',
```
Comment (alexanvl) [inline fr.js]: Language setting should be translated

File: ja.js [hunk 4]
```diff
@@ -7,6 +7,7 @@ export const ja = {
     fr: 'Français',
     es: 'Español',
     pt: 'Português',
+    vn: 'Vietnamese',
```
Comment (alexanvl) [inline ja.js]: Language setting should be translated

File: ko.js [hunk 5]
```diff
@@ -7,6 +7,7 @@ export const ko = {
     fr: 'Français',
     es: 'Español',
     pt: 'Português',
+    vn: 'Vietnamese',
```
Comment (alexanvl) [inline ko.js]: Language setting should be translated

File: pt.js [hunk 6]
```diff
@@ -7,6 +7,7 @@ export const pt = {
     fr: 'Français',
     es: 'Español',
     pt: 'Português',
+    vn: 'Vietnamese',
```
Comment (alexanvl) [inline pt.js]: Language setting should be translated

File: zh.js [hunk 7]
```diff
@@ -7,6 +7,7 @@ export const zh = {
     fr: 'Français',
     es: 'Español',
     pt: 'Português',
+    vn: 'Vietnamese',
```
Comment (alexanvl) [inline zh.js]: Language setting should be translated

File: vn.js [hunk 8]
```diff
@@ -0,0 +1,349 @@
+export const vn = {
+    translation: {
+        en: 'Tiếng Anh',
```
Comment (alexanvl) [inline vn.js]: Individual language settings should be translated to the local language

**Review by alexanvl — CHANGES_REQUESTED** [review]
Please fix the language settings translations and I will approve and merge this for release. Thank you!

**Review by alexanvl — APPROVED** [review]

**Pull request merged**